composeml/label_times.py

```diff
@@ -37,6 +37,24 @@ def __init__(self,
         self.settings = settings or {}
         self.settings['label_type'] = self.label_type
 
+    def __finalize__(self, other, method=None, **kwargs):
+        """Propagate metadata from other label times.
+
+        Args:
+            other (LabelTimes) : The label times from which to get the attributes from.
+            method (str) : A passed method name for optionally taking different types of propagation actions based on this value.
+        """
+        if method == 'concat':
```

Comment:
any reason to do this only for concat?

Reply:
So far, `concat` is the only method that I came across that doesn't propagate metadata -- other methods works fine.

Reply:
ok. we can leave it with just concat for now